parallel-install/pkg/prerequisites/prerequisites.go

```diff
@@ -35,7 +38,7 @@ func UninstallPrerequisites(ctx context.Context, prerequisites []components.Comp
 		}
 		err := prereq.UninstallComponent()
 		if err != nil {
-			return fmt.Errorf("Error uninstalling prerequisite %s: %v", prereq.Name, err)
+			log.Printf("%s Error uninstalling prerequisite %s: %v", logPrefix, prereq.Name, err)
```

Comment:
Maybe add information here that the uninstallation continues anyway?

Reply:
Done.